test/Y2ModuleConfig_test.rb

```diff
@@ -22,7 +43,7 @@
         [
           "audit-laf", "autofs", "ca_mgm", "firstboot", "language", "restore",
           "runlevel", "sshd", "sysconfig", "unknown_profile_item_1",
-          "unknown_profile_item_2", "users"
```

Comment:
Yes, but then you will have an unusable system because you cannot login. While the autoyast installation all user settings will be written in the second stage. 
The belonging autyast sections users,groups,user_defaults,login_settings are defined in
users.desktop:
X-SuSE-YaST-AutoInstMerge=users,groups,user_defaults,login_settings